src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallSummaryGenerator.kt

```diff
@@ -27,39 +31,57 @@ class RecallSummaryGenerator(
     Context().apply {
       val createdDate = LocalDate.now(clock).format(dateFormatter)
       val createdTime = ZonedDateTime.now(clock).withZoneSameInstant(ZoneId.of("Europe/London")).format(timeFormatter)
-      val firstAndMiddleNames = String.format("%s %s", context.prisoner.firstName ?: "", context.prisoner.middleNames ?: "").trim()
       setVariable("createdDate", createdDate)
       setVariable("createdTime", createdTime)
 
-      setVariable("mappaLevel1", context.recall.mappaLevel?.equals(MappaLevel.LEVEL_1) ?: false)
-      setVariable("mappaLevel2", context.recall.mappaLevel?.equals(MappaLevel.LEVEL_2) ?: false)
-      setVariable("mappaLevel3", context.recall.mappaLevel?.equals(MappaLevel.LEVEL_3) ?: false)
+      // TODO: MD What do we do if any of these values are not NULL?  They should all be present and valid
```

Comment:
I think you mean "if any of these values are NULL" rather than "if any of these values are not NULL"?
Comments, eh?  ;-).

Reply:
lol true, I'll fix in my next PR